backend/cmd/headlamp.go

```diff
@@ -54,24 +59,74 @@ func (h spaHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	path = strings.TrimLeft(path, h.baseURL)
+
 	// prepend the path with the path to the static directory
 	path = filepath.Join(h.staticPath, path)
 
 	// check whether a file exists at the given path
 	_, err = os.Stat(path)
 	if os.IsNotExist(err) {
 		// file does not exist, serve index.html
-		http.ServeFile(w, r, filepath.Join(h.staticPath, h.indexPath))
+		staticPath := filepath.Join(h.staticPath, h.indexPath)
+		http.ServeFile(w, r, staticPath)
+
```

Comment:
This change doesn't seem necessary/relevant to the commit.

Reply:
Moved it back to how it was.